Branch-SDK/src/io/branch/referral/ServerRequestInitSession.java

```diff
@@ -71,6 +71,26 @@ public boolean handleBranchViewIfAvailable(ServerResponse resp) {
         return isBranchViewShowing;
     }
 
+    @Override
+    public void onRequestSucceeded(ServerResponse response, Branch branch) {
+        // Check for any Third party SDK for data handling
+        String appOrigin = "";
+        try {
+            if (response.getObject().has(Defines.Jsonkey.AppOrigin.getKey())) {
+                appOrigin = response.getObject().getString(Defines.Jsonkey.AppOrigin.getKey());
+            }
+            boolean isFabricEnabled = appOrigin.equals(Defines.Jsonkey.AppOriginTwitterFabric.getKey());
```

Comment:
Hey Austin, we definitely want this to work for existing Branch/Fabric customers without requiring any extra work on their part. We're big believers in "it just works". Can we remove this or should we get on a call to discuss?